Contest_API.md

```diff
@@ -2108,6 +2112,33 @@ tags below are used.
 | accepted-winner         | A submission was accepted that changed the set of teams currently in the lead.
 | accepted-\<award>       | A submission was accepted that changed the set of teams awarded \<award>. Note that the above 4 are special cases of this.
 
+#### Modifying commentary 
+
+To add a commentary message, clients can use the `POST` method on the commentary endpoint.
+The following [capability](#capabilities) is required to add commentary,
+with description below:
+
+| Name               | Description
+| :----------------- | :----------
+| post\_comment      | POST a commentary message
+
+All requests must include a valid JSON object with the same properties as the commentary
+endpoint returns from a `GET` request with the following exceptions:
+
+- The `post\_comment` capability only has access to `POST`. `id`, `time`,
```

Comment:
I think this first sentence should be removed. In the clarifications endpoint it made sense since there was also a `{proxy,admin}_clar` capability with PUT access, but here there is only POST.
```suggestion
- `id`, `time`,
```